packages/client/test/routings.spec.ts

```diff
@@ -47,39 +61,50 @@ describe('libp2p content-routing', () => {
       throw new Error('ContentRouting not found')
     }
 
-    const providers = [{
-      Protocol: 'transport-bitswap',
-      Schema: 'bitswap',
-      Metadata: 'gBI=',
-      ID: (await generateKeyPair('Ed25519')).publicKey.toString(),
-      Addrs: ['/ip4/41.41.41.41/tcp/1234']
-    }, {
-      Protocol: 'transport-bitswap',
-      Schema: 'peer',
-      Metadata: 'gBI=',
-      ID: (await generateKeyPair('Ed25519')).publicKey.toString(),
-      Addrs: ['/ip4/42.42.42.42/tcp/1234']
-    }, {
-      ID: (await generateKeyPair('Ed25519')).publicKey.toString(),
-      Addrs: ['/ip4/43.43.43.43/tcp/1234']
-    }]
+    const providers = [
+      {
+        Protocol: 'transport-bitswap',
+        Schema: 'bitswap',
+        Metadata: 'gBI=',
+        ID: (await generateKeyPair('Ed25519')).publicKey.toString(),
+        Addrs: ['/ip4/41.41.41.41/tcp/1234']
+      },
+      {
+        Protocol: 'transport-bitswap',
+        Schema: 'peer',
+        Metadata: 'gBI=',
+        ID: (await generateKeyPair('Ed25519')).publicKey.toString(),
+        Addrs: ['/ip4/42.42.42.42/tcp/1234']
+      },
+      {
+        ID: (await generateKeyPair('Ed25519')).publicKey.toString(),
+        Addrs: ['/ip4/43.43.43.43/tcp/1234']
+      }
+    ]
```

Comment:
It looks like this change is style only, and unnecessary. 

I would highly recommend disabling editorconfig and auto styling that is not using `eslint` or aegir. If you disable auto-save, you should be able to run `npm run lint -- --fix` to fix issues until you get your editor to a good place.

FYI: my editor is vscode, and I have https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint installed